provision/provision-site.sh

```diff
@@ -84,6 +84,11 @@ function get_primary_host() {
 function vvv_provision_site_nginx_config() {
   local SITE_NAME=$1
   local SITE_NGINX_FILE=$2
+  local NGINX_CONFIGS=$(find "${VM_DIR}" -maxdepth 3 -name 'vvv-nginx.conf');
+  if [[ -z $NGINX_CONFIGS ]] ; then
+      vvv_error " ! Error: No nginx config was found, VVV will not know how to serve this site"
+      exit 1
+  fi
```

Comment:
I'm confused by this change, it's not the job of `vvv_provision_site_nginx_config` to locate the nginx config, it's passed to it as the second parameter. That task is the responsibility of a different function

Reply:
`vvv_provision_site_nginx` is the function that finds the Nginx file, and when it finds it it calls this function. That's what needed to be modified because when that function does not find a config, it fails the provisioner. The goal of the ticket is that it does not fail the provisioner but instead falls back to a default config